phpstan.neon

```diff
@@ -12,7 +12,19 @@ parameters:
 
     ignoreErrors:
         # traits
-        - '#Call to an undefined method Knp\\DoctrineBehaviors(.*?)#'
+        - '#Call to an undefined method Knp\\DoctrineBehaviors\\Contract\\Provider\\UserProviderInterface::changeUser\(\)#'
+        - '#Call to an undefined method Knp\\DoctrineBehaviors\\Contract\\Entity\\SluggableInterface::getId\(\)#'
+        - '#Call to an undefined method Knp\\DoctrineBehaviors\\Contract\\Entity\\TreeNodeInterface::getId\(\)#'
+        - '#Call to an undefined method Knp\\DoctrineBehaviors\\Contract\\Entity\\TreeNodeInterface::to(Flat)?Array\(\)#'
+
+        # translation proxy
+        - message: '#Call to an undefined method Knp\\DoctrineBehaviors\\Tests\\Fixtures\\Entity\\TranslatableEntity::(get|set)Title\(\)#'
+          path: tests/ORM/TranslatableTest.php
+
+        # translation inheritance, I don't wanted to change (Abstract|Extended)TranslationEntity(Translation)? classes
```

Comment:
Prevent this issue: 
![Capture d’écran de 2020-02-04 07-25-14](https://user-images.githubusercontent.com/2103975/73719368-876f5480-471f-11ea-9aba-f0503913cd2f.png)

Using `@implements` on abstract classes/children classes does not work very well. To make it work, some modifications are necessary but they are not pertinent and wanted I guess:
- `AbstractTranslatableEntity` should not implement `TranslatableInterface` and use trait `TranslatableTrait`
- `AbstractTranslatableEntityTranslation` should not implement `TranslationInterface` and use trait `TranslationTrait`
- `ExtendedTranslatableEntity` should implement `TranslatableInterface`, use trait `TranslatableTrait` and add `@implements` tag
- `ExtendedTranslatableEntityTranslation` should implement `TranslationInterface`, use trait `TranslationTrait` and add `@implements` tag

And so... inheritance now become useless :sweat_smile: